app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileState.kt

```diff
@@ -35,6 +52,12 @@ data class OtherUserProfileState(
     }
 }
 
+sealed class BottomSheetContent {
+    data class Conversation(val conversationData: ConversationSheetContent) : BottomSheetContent()
+    data class Mute(val conversationData: ConversationSheetContent) : BottomSheetContent()
+    data class ChangeRole(val groupState: OtherUserProfileGroupState?) : BottomSheetContent()
```

Comment:
Shouldn't `OtherUserProfileGroupState` also be non-nullable?

Reply:
true :)